### PR TITLE
Add support for Windows scene understanding as a spatial observer #6928

### PR DESCRIPTION
**Pull request by pinkwerks**
## Overview

Implements Scene Understanding as a Spatial Awareness observer.

Documentation and demo scene in:
Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/README.md
Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/DemoSpatialAwareness.unity

### CODE REVIEW

File: Assets/MixedRealityToolkit/Definitions/SpatialAwareness/SpatialAwarenessSurfaceTypes.cs [hunk 1]
```diff
@@ -36,6 +36,24 @@ public enum SpatialAwarenessSurfaceTypes
...
         /// requiring a horizontal surface.
         /// </remarks>
-        Platform = 1 << 4
+        Platform = 1 << 4,
+
+        /// <summary>
+        /// A surface that isn't a Platform but are known as objects (not unknown)
+        /// </summary>
+        /// <remarks>
+        /// These objets may be windows, monitors, stairs, etc.
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Definitions/SpatialAwareness/SpatialAwarenessSurfaceTypes.cs]:
```suggestion
        /// These objects may be windows, monitors, stairs, etc.
```

File: Assets/MixedRealityToolkit 1.Generated.meta [hunk 2]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: a0a6ba4d50f4c7d4aa014a0c75ebcd6b
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit 1.Generated.meta]: Please revert this add

**Comment by pinkwerks** [comment]
Closed to resubmit as draft

File: Assets/MixedRealityToolkit.Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset [hunk 3]
```diff
@@ -39,8 +39,8 @@ MonoBehaviour:
     Asset: {fileID: 102900000, guid: 63a00118e809c754f9c7911bb85d635f, type: 3}
   - Name: HandInteractionExamples
     Path: Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Scenes/HandInteractionExamples.unity
-    Included: 1
-    BuildIndex: 0
+    Included: 0
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Examples/Experimental/ExamplesHub/Profiles/MRTKExamplesHubSceneSystemProfile.asset]: Please revert this change

File: Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/.Documentation/BytesInDevicePortal.png.meta [hunk 4]
```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: f5d74b997f52d224ea01b4b80ffd492e
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/.Documentation/BytesInDevicePortal.png.meta]: We've typically stuck docs images in the Images folder in the root Documentation folder

**Comment by keveleigh** [comment]
Oh whoops, just saw this was closed. I'll re-review on the draft
